src/interface/IWrappedETH.sol

```diff
@@ -6,5 +6,5 @@ pragma solidity 0.8.17;
 /// @notice Wrapped ETH (version 9)
 /// @dev check https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code
 interface IWrappedETH {
-    function withdraw(uint wad) external;
+    function withdraw(uint256 wad) external;
```

Comment:
not sure why linter added this

Reply:
it's considered correct to not use the shorthand `uint`